views/update_project.erb

```diff
@@ -26,6 +26,10 @@
       <label for="webhookURL">Webhook URL</label>
       <input type="text" name="webhook_url" class="form-control"  id="webhookURL" value="<%= @project.webhook_url %>">
     </div>
+    <div class="form-group">
+      <label for="inputLineNotifyAccessToken">LINE Notify Access Token</label>
+      <input type="password" name="line_notify_access_token" class="form-control"  id="inputLineNotifyAccessToken" value="<% if @project.line_notify_access_token %>***<% end %>">
```

Comment:
This part is a little trickier if you want to show that it is filled in, without overwriting the token with "***" each time they save.

For starters, you could try to use the placeholder attribute ```<% if @project.line_notify_access_token %>placeholder="***"><% end %>``` to show that it is set or not. You will also likely need a button or checkbox to detect between when you want to clear the password and when you want to update the password. I will try to write up a quick example in a bit.